src/ChangelogsPlugin.php

```diff
@@ -98,25 +98,22 @@ public function postUpdate()
     }
 
     /**
-     * This method ensures all the classes required to make the plugin working
+     * This method ensures all the classes required to make the plugin work
      * are loaded.
      *
-     * It's required to avoid composer looking for classes no longer existing
-     * (after the plugin is updated or removed for example).
+     * It's required to avoid composer looking for classes which no longer exist
+     * (for example after the plugin is updated).
      *
      * Lot of classes (like operation handlers, url generators, Outputter, etc)
      * do not need this because they are already autoloaded at the activation
      * of the plugin.
      */
     private function autoloadNeededClasses()
     {
-        $classes = [
-            'Pyrech\ComposerChangelogs\Version',
-        ];
-
-        foreach ($classes as $class) {
-            // Force the class to be autoloaded
-            class_exists($class, true);
+        foreach (new \RecursiveIteratorIterator(new \RecursiveDirectoryIterator(__DIR__, \FilesystemIterator::SKIP_DOTS)) as $file) {
+            if ('.php' === substr($file, 0, -4)) {
+                require_once $file;
```

Comment:
```suggestion
                class_exists(__NAMESPACE__.str_replace('/', '\\', substr($file, \strlen(__DIR__), -4)));
```
May I just ask you to change to the latest version used in Flex (aka https://github.com/symfony/flex/blob/master/src/Flex.php#L97-L103)